docs/archive/LangRefNew.rst

```diff
@@ -1348,7 +1348,7 @@ otherwise empty cases in switch statements.
 
 ::
 
-  func classifyPoint(point: (Int, Int)) {
```

Comment:
It's not actually a change in API, but I don't think changing the doc for API naming is in scope for this change.

Though I'm not against this change if you wanna just take it.